cloud.foundry.cli/src/main/java/cloud/foundry/cli/services/UpdateController.java

```diff
@@ -80,42 +81,61 @@ public Integer call() throws Exception {
         @Option(names = { "-f", "--force" }, required = false, description = "Force deletion without confirmation.")
         Boolean force;
 
+        private Map<String, ServiceBean> readServicesFromYamlFile() throws IOException {
+            SpecBean specBean = YamlMapper.loadBean(yamlCommandOptions.getYamlFilePath(), SpecBean.class);
+            return specBean.getServices();
+        }
+
         @Override
         public Integer call() throws Exception {
+            Map<String, ServiceBean> services = readServicesFromYamlFile();
 
-            if (force != null) {
-                doRemoveServiceInstance(yamlCommandOptions);
-            } else {
+            // in case the --force flag is not specified, we shall prompt the user to prevent the removal of services
+            // as that might remove valuable data a swell
+            if (force == null) {
                 if (System.console() == null) {
                     Log.error("--force/-f not supplied and not running in terminal, aborting");
                     return 2;
                 }
 
-                System.out.println("Really delete the services y/n?");
+                System.out.print("Requested removal of the following services: ");
+                for (String serviceName : services.keySet()) {
+                    System.out.print("\"" + serviceName + "\" ");
+                }
+                System.out.println();
```

Comment:
You could delete this line and just write a '\n' in 

System.out.print("\nAre you sure you want to permanently delete those? [y|N] ");

Reply:
I like to rely on Java for platform specific questions. The question how to terminate a line is such a question, '\n' only works on Linux-ish platforms. Therefore it makes sense to use `println()`, even here.